README.md

```diff
@@ -164,6 +164,11 @@ cd haitaton-backend
 When running locally, the system emails are sent to smtp4dev, which is started as part of the Docker
 Compose setup. You can access the sent emails by opening http://localhost:3003.
 
+Creation of new emails is done with [mjml.io](https://mjml.io/). Either IntelliJ or Visual Studio Code plugin MJML is
```

Comment:
The IntelliJ IDEA plugin can't do the HTML export. We'll need more tooling to make the templates ergonomic, but that can be tackled later.

Reply:
IntelliJ plugin provides the html output that you can copy-paste. I did not want to write very detailed instructions here as we should probably have a script etc. to automate this (which is tackled later).